Branch-SDK/Branch-SDK/BNCDeviceInfo.m

```diff
@@ -124,7 +125,7 @@ + (NSString*) userAgentString {
             BNCPreferenceHelper *preferences = [BNCPreferenceHelper preferenceHelper];
             preferences.browserUserAgentString = browserUserAgentString;
             preferences.lastSystemBuildVersion = self.systemBuildVersion;
-			//NSLog(@"[Branch] userAgentString: '%@'.", browserUserAgentString);
```

Comment:
We recommend developers enable setDebug while integrating Branch. setDebug is not an internal debugging flag. We should only be logging messages that are useful to external developers and the user agent string is meaningless.